static/sass/styles-v1.scss

```diff
@@ -12,6 +12,13 @@
 // vendor
 @import 'vendors/global-nav/build';
 
+// patterns
```

Comment:
It's just an artefact of the code restructure that happened after this branch was made, I'll remove the old one.